src/drivers/eeprom_i2c.c

```diff
@@ -0,0 +1,135 @@
+#include "simba.h"
+#if CONFIG_PUMBAA_CLASS_I2C_SOFT == 1
+struct i2c_soft_driver_t i2c;
+
+int eeprom_i2c_init(struct i2c_soft_driver_t *self_p,

```

Comment:
Give an initialized i2c driver as argument instead of all its configuration.

https://github.com/eerimoq/simba/blob/master/src/drivers/sht3xd.h#L75-L77